book/src/requirements/apple.md

```diff
@@ -0,0 +1,64 @@
+# macOS & iOS Requirements
+
+Unless the "fips" feature is enabled, building aws-lc-rs v1.7.0 (or later) for the following targets should only require
+a C/C++ compiler.
+
+| Platform               | *default*              | **fips**                   | bindgen required? |
+|------------------------|------------------------|----------------------------|-------------------|
+| `aarch64-apple-darwin` | C/C++ Compiler         | C/C++ Compiler, CMake & Go | No                | 
+| `x86_64-apple-darwin`  | C/C++ Compiler         | C/C++ Compiler, CMake & Go | No                |
+| `aarch64-apple-ios`    | C/C++ Compiler & CMake | **Not Supported**          | **_Yes_**         |
+| `x86_64-apple-ios`     | C/C++ Compiler & CMake | **Not Supported**          | **_Yes_**         |
+| _Other_                | C/C++ Compiler & CMake | **Not Supported**          | **_Yes_**         |
```

Comment:
`_Other_` makes sense in the other platforms. Are there any other alternatives for apple not listed already?

Reply:
Good point.  I'm not aware of any other "apple" platforms that we could support.  I'll remove this line.